2025-04-29 01:53:43,924 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 01:53:43,925 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 01:53:43,926 - INFO - Started reloader process [465645] using WatchFiles
2025-04-29 01:53:45,160 - INFO - Started server process [465674]
2025-04-29 01:53:45,162 - INFO - Waiting for application startup.
2025-04-29 01:53:45,163 - INFO - Application startup complete.
2025-04-29 02:18:58,736 - INFO - Shutting down
2025-04-29 02:18:58,845 - INFO - Waiting for application shutdown.
2025-04-29 02:18:58,850 - INFO - Application shutdown complete.
2025-04-29 02:18:58,851 - INFO - Finished server process [465674]
2025-04-29 02:18:58,870 - INFO - Stopping reloader process [465645]
2025-04-29 02:19:06,512 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 02:19:06,513 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 02:19:06,514 - INFO - Started reloader process [478197] using WatchFiles
2025-04-29 02:19:07,721 - INFO - Started server process [478226]
2025-04-29 02:19:07,724 - INFO - Waiting for application startup.
2025-04-29 02:19:07,725 - INFO - Application startup complete.
2025-04-29 02:29:51,269 - INFO - Shutting down
2025-04-29 02:29:51,378 - INFO - Waiting for application shutdown.
2025-04-29 02:29:51,380 - INFO - Application shutdown complete.
2025-04-29 02:29:51,381 - INFO - Finished server process [478226]
2025-04-29 02:29:51,401 - INFO - Stopping reloader process [478197]
2025-04-29 02:37:39,576 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 02:37:39,577 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 02:37:39,577 - INFO - Started reloader process [484526] using WatchFiles
2025-04-29 02:37:40,756 - INFO - Started server process [484559]
2025-04-29 02:37:40,758 - INFO - Waiting for application startup.
2025-04-29 02:37:40,759 - INFO - Application startup complete.
2025-04-29 03:09:35,463 - INFO - ('127.0.0.1', 58206) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:09:35,467 - INFO - connection open
2025-04-29 03:09:35,469 - INFO - connection closed
2025-04-29 03:13:16,174 - INFO - ('127.0.0.1', 57580) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:13:16,176 - INFO - connection open
2025-04-29 03:13:16,178 - INFO - connection closed
2025-04-29 03:16:16,696 - INFO - ('127.0.0.1', 37280) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:16:16,698 - INFO - connection open
2025-04-29 03:16:16,702 - INFO - connection closed
2025-04-29 03:17:45,070 - INFO - ('127.0.0.1', 59732) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:17:45,072 - INFO - connection open
2025-04-29 03:17:45,074 - INFO - connection closed
2025-04-29 03:20:52,657 - INFO - ('127.0.0.1', 45756) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:20:52,661 - INFO - connection open
2025-04-29 03:20:52,664 - INFO - connection closed
2025-04-29 03:25:31,951 - INFO - ('127.0.0.1', 55152) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:25:31,954 - INFO - connection open
2025-04-29 03:25:31,957 - INFO - connection closed
2025-04-29 03:29:17,307 - INFO - ('127.0.0.1', 47312) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:29:17,309 - INFO - connection open
2025-04-29 03:29:17,310 - INFO - connection closed
2025-04-29 03:33:09,454 - INFO - ('127.0.0.1', 48488) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:33:09,459 - INFO - connection open
2025-04-29 03:33:09,464 - INFO - connection closed
2025-04-29 03:35:27,095 - INFO - ('127.0.0.1', 51422) - "WebSocket /train/ws?password=$2b$12$CFgpc4B60UhQ/144Sa9Eku.yPv6TJwH6F7f2mCsR4qjbCM3JVFNCq" [accepted]
2025-04-29 03:35:27,099 - INFO - connection open
2025-04-29 03:35:27,102 - INFO - connection closed
2025-04-29 03:37:01,976 - ERROR - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 76, in collapse_excgroups
  |     yield
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 174, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 772, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    |     with recv_stream, send_stream, collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    |     response = await handler(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    |     raise app_exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    |     await route.handle(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    |     response = await f(request)
    |                ^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    |     return BaseResponse(e.args[0]["message"]).error()
    |                         ~~~~~~~~~^^^^^^^^^^^
    | TypeError: string indices must be integers, not 'str'
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    with recv_stream, send_stream, collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    raise app_exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    return BaseResponse(e.args[0]["message"]).error()
                        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
2025-04-29 03:37:17,682 - INFO - Shutting down
2025-04-29 03:37:17,786 - INFO - Waiting for application shutdown.
2025-04-29 03:37:17,789 - INFO - Application shutdown complete.
2025-04-29 03:37:17,794 - INFO - Finished server process [484559]
2025-04-29 03:37:17,843 - INFO - Stopping reloader process [484526]
2025-04-29 03:37:24,492 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 03:37:24,493 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 03:37:24,494 - INFO - Started reloader process [513244] using WatchFiles
2025-04-29 03:37:25,642 - INFO - Started server process [513275]
2025-04-29 03:37:25,645 - INFO - Waiting for application startup.
2025-04-29 03:37:25,646 - INFO - Application startup complete.
2025-04-29 03:38:23,138 - INFO - ('127.0.0.1', 42616) - "WebSocket /train/ws?password=$2b$12$xMiDh9TFIlShBcr50KmWJuKfCNbS5p/N/q9.hwuKxS/DSuLm6xeHa" [accepted]
2025-04-29 03:38:23,146 - INFO - connection open
2025-04-29 03:38:23,152 - INFO - connection closed
2025-04-29 03:39:25,732 - INFO - Shutting down
2025-04-29 03:39:25,839 - INFO - Waiting for application shutdown.
2025-04-29 03:39:25,841 - INFO - Application shutdown complete.
2025-04-29 03:39:25,842 - INFO - Finished server process [513275]
2025-04-29 03:39:25,871 - INFO - Stopping reloader process [513244]
2025-04-29 03:39:36,638 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 03:39:36,639 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 03:39:36,639 - INFO - Started reloader process [514858] using WatchFiles
2025-04-29 03:39:37,898 - INFO - Started server process [514878]
2025-04-29 03:39:37,900 - INFO - Waiting for application startup.
2025-04-29 03:39:37,901 - INFO - Application startup complete.
2025-04-29 03:40:21,022 - INFO - ('127.0.0.1', 58140) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:40:21,024 - INFO - connection open
2025-04-29 03:40:21,026 - INFO - connection closed
2025-04-29 03:44:13,387 - INFO - ('127.0.0.1', 38594) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:44:13,391 - INFO - connection open
2025-04-29 03:44:13,393 - INFO - connection closed
2025-04-29 03:49:23,336 - INFO - ('127.0.0.1', 42176) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:49:23,340 - INFO - connection open
2025-04-29 03:49:23,342 - INFO - connection closed
2025-04-29 03:55:25,149 - INFO - ('127.0.0.1', 55422) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:55:25,152 - INFO - connection open
2025-04-29 03:55:25,155 - INFO - connection closed
2025-04-29 03:57:16,457 - INFO - ('127.0.0.1', 43538) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:57:16,460 - INFO - connection open
2025-04-29 03:57:16,464 - INFO - connection closed
2025-04-29 03:59:18,145 - INFO - ('127.0.0.1', 36314) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 03:59:18,150 - INFO - connection open
2025-04-29 03:59:18,154 - INFO - connection closed
2025-04-29 04:00:30,771 - INFO - ('127.0.0.1', 41580) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:00:30,774 - INFO - connection open
2025-04-29 04:00:30,778 - INFO - connection closed
2025-04-29 04:01:52,871 - INFO - ('127.0.0.1', 47298) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:01:52,878 - INFO - connection open
2025-04-29 04:01:52,882 - INFO - connection closed
2025-04-29 04:09:24,207 - INFO - ('127.0.0.1', 46090) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:09:24,213 - INFO - connection open
2025-04-29 04:09:24,216 - INFO - connection closed
2025-04-29 04:16:14,206 - INFO - ('127.0.0.1', 57718) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:16:14,207 - INFO - connection open
2025-04-29 04:16:14,209 - INFO - connection closed
2025-04-29 04:18:12,541 - INFO - ('127.0.0.1', 48602) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:18:12,542 - INFO - connection open
2025-04-29 04:18:12,545 - INFO - connection closed
2025-04-29 04:19:17,199 - INFO - ('127.0.0.1', 41214) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:19:17,201 - INFO - connection open
2025-04-29 04:19:17,203 - INFO - connection closed
2025-04-29 04:21:18,966 - INFO - ('127.0.0.1', 54650) - "WebSocket /train/ws?password=$2b$12$Hz2htHLSO5S6dMh3c8pliurFMU8LppMRdZCSucLTi2vqRkM0X31Qe" [accepted]
2025-04-29 04:21:18,968 - INFO - connection open
2025-04-29 04:21:18,971 - INFO - connection closed
2025-04-29 04:22:28,293 - INFO - Shutting down
2025-04-29 04:22:28,398 - INFO - Waiting for application shutdown.
2025-04-29 04:22:28,401 - INFO - Application shutdown complete.
2025-04-29 04:22:28,402 - INFO - Finished server process [514878]
2025-04-29 04:22:28,435 - INFO - Stopping reloader process [514858]
2025-04-29 04:22:34,736 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 04:22:34,737 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 04:22:34,737 - INFO - Started reloader process [539070] using WatchFiles
2025-04-29 04:22:36,123 - INFO - Started server process [539098]
2025-04-29 04:22:36,125 - INFO - Waiting for application startup.
2025-04-29 04:22:36,126 - INFO - Application startup complete.
2025-04-29 04:28:11,715 - INFO - ('127.0.0.1', 57004) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 04:28:11,722 - INFO - connection open
2025-04-29 04:28:11,725 - INFO - connection closed
2025-04-29 04:33:57,665 - INFO - ('127.0.0.1', 45174) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 04:33:57,668 - INFO - connection open
2025-04-29 04:33:57,670 - INFO - connection closed
2025-04-29 04:43:04,516 - INFO - ('127.0.0.1', 53724) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 04:43:04,519 - INFO - connection open
2025-04-29 04:43:04,522 - INFO - connection closed
2025-04-29 04:49:17,413 - INFO - ('127.0.0.1', 51860) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 04:49:17,415 - INFO - connection open
2025-04-29 04:49:17,419 - INFO - connection closed
2025-04-29 06:20:36,657 - INFO - ('127.0.0.1', 46150) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:20:36,658 - INFO - connection open
2025-04-29 06:20:36,660 - INFO - connection closed
2025-04-29 06:22:08,918 - INFO - ('127.0.0.1', 53418) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:22:08,920 - INFO - connection open
2025-04-29 06:22:08,925 - INFO - connection closed
2025-04-29 06:25:10,217 - INFO - ('127.0.0.1', 35288) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:25:10,220 - INFO - connection open
2025-04-29 06:25:10,225 - INFO - connection closed
2025-04-29 06:26:53,004 - INFO - ('127.0.0.1', 54982) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:26:53,007 - INFO - connection open
2025-04-29 06:26:53,008 - INFO - connection closed
2025-04-29 06:28:43,480 - INFO - ('127.0.0.1', 54088) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:28:43,481 - INFO - connection open
2025-04-29 06:28:43,482 - INFO - connection closed
2025-04-29 06:57:52,498 - INFO - ('127.0.0.1', 54454) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 06:57:52,506 - INFO - connection open
2025-04-29 06:57:52,508 - INFO - connection closed
2025-04-29 07:08:08,947 - INFO - ('127.0.0.1', 38514) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 07:08:08,949 - INFO - connection open
2025-04-29 07:08:08,951 - INFO - connection closed
2025-04-29 07:20:04,267 - INFO - ('127.0.0.1', 59550) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 07:20:04,269 - INFO - connection open
2025-04-29 07:20:04,271 - INFO - connection closed
2025-04-29 08:08:28,439 - INFO - ('127.0.0.1', 37140) - "WebSocket /train/ws?password=$2b$12$355NrClp/0THwSQ1asBrluMyiAMjSJPRTTt4w4tXVycC3YdCDPxPS" [accepted]
2025-04-29 08:08:28,463 - INFO - connection open
2025-04-29 08:08:28,472 - INFO - connection closed
2025-04-29 08:42:25,682 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-04-29 08:42:25,685 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-04-29 08:42:25,686 - INFO - Started reloader process [6291] using WatchFiles
2025-04-29 08:42:26,732 - INFO - Started server process [6317]
2025-04-29 08:42:26,734 - INFO - Waiting for application startup.
2025-04-29 08:42:26,735 - INFO - Application startup complete.
2025-04-29 09:22:58,494 - INFO - ('127.0.0.1', 50048) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 09:22:58,500 - INFO - connection open
2025-04-29 09:22:58,503 - INFO - connection closed
2025-04-29 09:28:56,755 - INFO - ('127.0.0.1', 41686) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 09:28:56,759 - INFO - connection open
2025-04-29 09:28:56,966 - INFO - connection closed
2025-04-29 09:50:14,752 - INFO - ('127.0.0.1', 45844) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 09:50:14,753 - INFO - connection open
2025-04-29 09:50:14,959 - INFO - connection closed
2025-04-29 09:51:38,556 - INFO - ('127.0.0.1', 44612) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 09:51:38,557 - INFO - connection open
2025-04-29 09:51:38,559 - INFO - connection closed
2025-04-29 09:55:54,686 - INFO - ('127.0.0.1', 60646) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 09:55:54,687 - INFO - connection open
2025-04-29 09:55:54,692 - INFO - connection closed
2025-04-29 10:33:17,671 - INFO - ('127.0.0.1', 51386) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-04-29 10:33:17,676 - INFO - connection open
2025-04-29 10:33:17,681 - INFO - connection closed
2025-05-04 10:05:37,198 - INFO - ('127.0.0.1', 40128) - "WebSocket /train/ws?password=$2b$12$v3DimUShXJG8phJQiH1PwOF89pScvUuH9mytphfstu9JHLb37LW4K" [accepted]
2025-05-04 10:05:37,234 - INFO - connection open
2025-05-04 10:05:37,252 - INFO - connection closed
2025-05-04 10:11:49,055 - INFO - Shutting down
2025-05-04 10:11:49,160 - INFO - Waiting for application shutdown.
2025-05-04 10:11:49,162 - INFO - Application shutdown complete.
2025-05-04 10:11:49,169 - INFO - Finished server process [6317]
2025-05-04 10:11:49,202 - INFO - Stopping reloader process [6291]
2025-05-04 10:11:56,101 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-04 10:11:56,103 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-04 10:11:56,103 - INFO - Started reloader process [120439] using WatchFiles
2025-05-04 10:11:57,238 - INFO - Started server process [120469]
2025-05-04 10:11:57,240 - INFO - Waiting for application startup.
2025-05-04 10:11:57,241 - INFO - Application startup complete.
2025-05-04 10:48:30,783 - INFO - ('127.0.0.1', 38520) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 10:48:30,789 - INFO - connection open
2025-05-04 10:48:30,793 - INFO - connection closed
2025-05-04 11:54:36,912 - INFO - ('127.0.0.1', 51960) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 11:54:36,929 - INFO - connection open
2025-05-04 11:54:36,953 - INFO - connection closed
2025-05-04 13:35:34,220 - INFO - ('127.0.0.1', 60270) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 13:35:34,232 - INFO - connection open
2025-05-04 13:35:34,238 - INFO - connection closed
2025-05-04 15:44:06,681 - INFO - ('127.0.0.1', 40288) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 15:44:06,693 - INFO - connection open
2025-05-04 15:44:06,698 - INFO - connection closed
2025-05-04 16:23:53,527 - INFO - ('127.0.0.1', 35318) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 16:23:53,533 - INFO - connection open
2025-05-04 16:23:53,537 - INFO - connection closed
2025-05-04 17:12:34,184 - INFO - ('127.0.0.1', 37978) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-04 17:12:34,191 - INFO - connection open
2025-05-04 17:12:34,195 - INFO - connection closed
2025-05-05 02:24:43,947 - INFO - ('127.0.0.1', 38452) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-05 02:24:43,961 - INFO - connection open
2025-05-05 02:24:43,970 - INFO - connection closed
2025-05-05 03:14:22,334 - INFO - ('127.0.0.1', 40216) - "WebSocket /train/ws?password=$2b$12$L279h4vy2D1VYoRGxqBuBO30.M8LUjxidyXVEEBP.J05adQyHy2F6" [accepted]
2025-05-05 03:14:22,340 - INFO - connection open
2025-05-05 03:14:22,344 - INFO - connection closed
2025-05-05 03:15:21,729 - INFO - Shutting down
2025-05-05 03:15:21,833 - INFO - Waiting for application shutdown.
2025-05-05 03:15:21,835 - INFO - Application shutdown complete.
2025-05-05 03:15:21,837 - INFO - Finished server process [120469]
2025-05-05 03:15:21,849 - INFO - Stopping reloader process [120439]
2025-05-05 03:15:28,983 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-05 03:15:28,984 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-05 03:15:28,985 - INFO - Started reloader process [382975] using WatchFiles
2025-05-05 03:15:30,108 - INFO - Started server process [383005]
2025-05-05 03:15:30,110 - INFO - Waiting for application startup.
2025-05-05 03:15:30,111 - INFO - Application startup complete.
2025-05-05 03:23:04,404 - INFO - ('127.0.0.1', 47782) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:23:04,406 - INFO - connection open
2025-05-05 03:23:04,408 - INFO - connection closed
2025-05-05 03:25:46,457 - INFO - ('127.0.0.1', 46416) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:25:46,458 - INFO - connection open
2025-05-05 03:25:46,460 - INFO - connection closed
2025-05-05 03:28:10,211 - INFO - ('127.0.0.1', 55822) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:28:10,214 - INFO - connection open
2025-05-05 03:28:10,219 - INFO - connection closed
2025-05-05 03:30:44,156 - INFO - ('127.0.0.1', 51878) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:30:44,160 - INFO - connection open
2025-05-05 03:30:44,170 - INFO - connection closed
2025-05-05 03:33:13,074 - INFO - ('127.0.0.1', 55636) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:33:13,079 - INFO - connection open
2025-05-05 03:33:13,083 - INFO - connection closed
2025-05-05 03:35:43,364 - INFO - ('127.0.0.1', 54428) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:35:43,372 - INFO - connection open
2025-05-05 03:35:43,376 - INFO - connection closed
2025-05-05 03:43:07,303 - INFO - ('127.0.0.1', 59036) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:43:07,306 - INFO - connection open
2025-05-05 03:43:07,309 - INFO - connection closed
2025-05-05 03:48:36,631 - INFO - ('127.0.0.1', 42836) - "WebSocket /train/ws?password=$2b$12$H6hf9JyAwRMdv3A6xcajJOFdYcRb29AZ88G0zn5MgfIOrYpBiqPgC" [accepted]
2025-05-05 03:48:36,633 - INFO - connection open
2025-05-05 03:48:36,636 - INFO - connection closed
2025-05-05 03:53:50,292 - INFO - Shutting down
2025-05-05 03:53:50,399 - INFO - Waiting for application shutdown.
2025-05-05 03:53:50,400 - INFO - Application shutdown complete.
2025-05-05 03:53:50,401 - INFO - Finished server process [383005]
2025-05-05 03:53:50,418 - INFO - Stopping reloader process [382975]
2025-05-05 03:53:54,789 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-05 03:53:54,790 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-05 03:53:54,791 - INFO - Started reloader process [403064] using WatchFiles
2025-05-05 03:53:56,079 - INFO - Started server process [403101]
2025-05-05 03:53:56,082 - INFO - Waiting for application startup.
2025-05-05 03:53:56,082 - INFO - Application startup complete.
2025-05-05 03:54:47,505 - INFO - ('127.0.0.1', 35110) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 03:54:47,510 - INFO - connection open
2025-05-05 03:54:47,513 - INFO - connection closed
2025-05-05 03:59:04,680 - INFO - ('127.0.0.1', 58520) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 03:59:04,688 - INFO - connection open
2025-05-05 03:59:04,693 - INFO - connection closed
2025-05-05 04:01:54,733 - INFO - ('127.0.0.1', 57012) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 04:01:54,748 - INFO - connection open
2025-05-05 04:01:54,786 - INFO - connection closed
2025-05-05 04:08:45,457 - INFO - ('127.0.0.1', 49184) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 04:08:45,473 - INFO - connection open
2025-05-05 04:08:45,478 - INFO - connection closed
2025-05-05 05:16:17,670 - INFO - ('127.0.0.1', 56976) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 05:16:17,678 - INFO - connection open
2025-05-05 05:16:17,682 - INFO - connection closed
2025-05-05 07:04:01,267 - INFO - ('127.0.0.1', 38860) - "WebSocket /train/ws?password=$2b$12$H20NtbP61u9J9KZNcKnNVeO9t4zsj6zJXnjyZAOJdTPxziK6gYjCO" [accepted]
2025-05-05 07:04:01,278 - INFO - connection open
2025-05-05 07:04:01,285 - INFO - connection closed
2025-05-05 07:09:34,263 - INFO - Shutting down
2025-05-05 07:09:34,371 - INFO - Waiting for application shutdown.
2025-05-05 07:09:34,373 - INFO - Application shutdown complete.
2025-05-05 07:09:34,373 - INFO - Finished server process [403101]
2025-05-05 07:09:34,404 - INFO - Stopping reloader process [403064]
2025-05-05 07:09:46,854 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-05 07:09:46,855 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-05 07:09:46,857 - INFO - Started reloader process [506940] using WatchFiles
2025-05-05 07:09:48,235 - INFO - Started server process [506966]
2025-05-05 07:09:48,238 - INFO - Waiting for application startup.
2025-05-05 07:09:48,238 - INFO - Application startup complete.
2025-05-05 08:24:11,880 - INFO - ('127.0.0.1', 35038) - "WebSocket /train/ws?password=$2b$12$BDPvgK/Nwdoq7ca.Ft8M1OF4xnWstG/PNFs/Jvob34vxlTKFiMa3i" [accepted]
2025-05-05 08:24:11,900 - INFO - connection open
2025-05-05 08:24:11,908 - INFO - connection closed
2025-05-05 08:41:23,859 - INFO - Shutting down
2025-05-05 08:41:23,966 - INFO - Waiting for application shutdown.
2025-05-05 08:41:23,973 - INFO - Application shutdown complete.
2025-05-05 08:41:23,977 - INFO - Finished server process [506966]
2025-05-05 08:41:24,011 - INFO - Stopping reloader process [506940]
2025-05-05 08:41:31,004 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-05 08:41:31,005 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-05 08:41:31,006 - INFO - Started reloader process [555406] using WatchFiles
2025-05-05 08:41:32,362 - INFO - Started server process [555437]
2025-05-05 08:41:32,364 - INFO - Waiting for application startup.
2025-05-05 08:41:32,364 - INFO - Application startup complete.
2025-05-05 09:57:55,068 - INFO - ('127.0.0.1', 42788) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 09:57:55,076 - INFO - connection open
2025-05-05 09:57:55,080 - INFO - connection closed
2025-05-05 13:21:47,874 - INFO - ('127.0.0.1', 57230) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 13:21:47,884 - INFO - connection open
2025-05-05 13:21:47,889 - INFO - connection closed
2025-05-05 14:02:04,967 - INFO - ('127.0.0.1', 55960) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 14:02:04,974 - INFO - connection open
2025-05-05 14:02:04,976 - INFO - connection closed
2025-05-05 14:33:45,867 - INFO - ('127.0.0.1', 41676) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 14:33:45,873 - INFO - connection open
2025-05-05 14:33:45,878 - INFO - connection closed
2025-05-05 14:49:31,740 - INFO - ('127.0.0.1', 39506) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 14:49:31,749 - INFO - connection open
2025-05-05 14:49:31,758 - INFO - connection closed
2025-05-05 15:00:20,062 - INFO - ('127.0.0.1', 39730) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 15:00:20,070 - INFO - connection open
2025-05-05 15:00:20,075 - INFO - connection closed
2025-05-05 15:11:54,780 - ERROR - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 76, in collapse_excgroups
  |     yield
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 174, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 772, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    |     with recv_stream, send_stream, collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    |     response = await handler(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    |     raise app_exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    |     await route.handle(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    |     response = await f(request)
    |                ^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    |     return BaseResponse(e.args[0]["message"]).error()
    |                         ~~~~~~~~~^^^^^^^^^^^
    | TypeError: string indices must be integers, not 'str'
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    with recv_stream, send_stream, collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    raise app_exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    return BaseResponse(e.args[0]["message"]).error()
                        ~~~~~~~~~^^^^^^^^^^^
TypeError: string indices must be integers, not 'str'
2025-05-05 15:25:29,503 - INFO - ('127.0.0.1', 54646) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-05 15:25:29,513 - INFO - connection open
2025-05-05 15:25:29,520 - INFO - connection closed
2025-05-06 01:55:05,523 - INFO - ('127.0.0.1', 34738) - "WebSocket /train/ws?password=$2b$12$jngI6x6VgAF67WTq8pDjsOC6yxwebfPkyJjp6vcwv4D1J5FXcbqyi" [accepted]
2025-05-06 01:55:05,530 - INFO - connection open
2025-05-06 01:55:05,536 - INFO - connection closed
2025-05-06 02:48:44,371 - INFO - Shutting down
2025-05-06 02:48:44,476 - INFO - Waiting for application shutdown.
2025-05-06 02:48:44,478 - INFO - Application shutdown complete.
2025-05-06 02:48:44,479 - INFO - Finished server process [555437]
2025-05-06 02:48:44,500 - INFO - Stopping reloader process [555406]
2025-05-06 02:48:49,759 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-06 02:48:49,760 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-06 02:48:49,761 - INFO - Started reloader process [735184] using WatchFiles
2025-05-06 02:48:50,893 - INFO - Started server process [735205]
2025-05-06 02:48:50,895 - INFO - Waiting for application startup.
2025-05-06 02:48:50,896 - INFO - Application startup complete.
2025-05-06 03:05:16,148 - INFO - ('127.0.0.1', 55152) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 03:05:16,168 - INFO - connection open
2025-05-06 03:05:16,171 - INFO - connection closed
2025-05-06 08:09:08,552 - INFO - ('127.0.0.1', 49680) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:09:08,555 - INFO - connection open
2025-05-06 08:09:08,558 - INFO - connection closed
2025-05-06 08:18:57,325 - INFO - ('127.0.0.1', 57760) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:18:57,343 - INFO - connection open
2025-05-06 08:18:57,346 - INFO - connection closed
2025-05-06 08:32:05,204 - INFO - ('127.0.0.1', 54366) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:32:05,211 - INFO - connection open
2025-05-06 08:32:05,216 - INFO - connection closed
2025-05-06 08:35:37,894 - INFO - ('127.0.0.1', 49290) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:35:37,896 - INFO - connection open
2025-05-06 08:35:37,897 - INFO - connection closed
2025-05-06 08:41:04,601 - INFO - ('127.0.0.1', 38084) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:41:04,613 - INFO - connection open
2025-05-06 08:41:04,617 - INFO - connection closed
2025-05-06 08:51:23,988 - INFO - ('127.0.0.1', 59844) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 08:51:23,996 - INFO - connection open
2025-05-06 08:51:24,011 - INFO - connection closed
2025-05-06 09:00:55,983 - INFO - ('127.0.0.1', 37490) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 09:00:55,994 - INFO - connection open
2025-05-06 09:00:56,009 - INFO - connection closed
2025-05-06 09:08:47,916 - INFO - ('127.0.0.1', 58010) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 09:08:47,921 - INFO - connection open
2025-05-06 09:08:47,923 - INFO - connection closed
2025-05-06 09:21:02,633 - INFO - ('127.0.0.1', 43934) - "WebSocket /train/ws?password=$2b$12$0uxpOz5p6pD4547ZhhhVPubGQefzXelav0quBSYB7Pkq1grqNJc1C" [accepted]
2025-05-06 09:21:02,642 - INFO - connection open
2025-05-06 09:21:02,645 - INFO - connection closed
2025-05-07 01:58:51,036 - INFO - Shutting down
2025-05-07 01:58:51,142 - INFO - Waiting for application shutdown.
2025-05-07 01:58:51,145 - INFO - Application shutdown complete.
2025-05-07 01:58:51,146 - INFO - Finished server process [735205]
2025-05-07 01:58:51,201 - INFO - Stopping reloader process [735184]
2025-05-07 01:59:08,426 - INFO - Will watch for changes in these directories: ['/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api']
2025-05-07 01:59:08,427 - INFO - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-05-07 01:59:08,428 - INFO - Started reloader process [930508] using WatchFiles
2025-05-07 01:59:09,616 - INFO - Started server process [930553]
2025-05-07 01:59:09,618 - INFO - Waiting for application startup.
2025-05-07 01:59:09,618 - INFO - Application startup complete.
2025-05-07 02:16:01,458 - INFO - ('127.0.0.1', 53158) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 02:16:01,464 - INFO - connection open
2025-05-07 02:16:01,468 - INFO - connection closed
2025-05-07 02:30:40,253 - INFO - ('127.0.0.1', 58704) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 02:30:40,260 - INFO - connection open
2025-05-07 02:30:40,275 - INFO - connection closed
2025-05-07 02:37:48,786 - INFO - ('127.0.0.1', 48578) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 02:37:48,789 - INFO - connection open
2025-05-07 02:37:48,793 - INFO - connection closed
2025-05-07 02:42:02,986 - INFO - ('127.0.0.1', 45796) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 02:42:02,993 - INFO - connection open
2025-05-07 02:42:02,998 - INFO - connection closed
2025-05-07 02:53:57,531 - INFO - ('127.0.0.1', 35554) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 02:53:57,558 - INFO - connection open
2025-05-07 02:53:57,569 - INFO - connection closed
2025-05-07 03:10:39,724 - INFO - ('127.0.0.1', 39616) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 03:10:39,757 - INFO - connection open
2025-05-07 03:10:39,763 - INFO - connection closed
2025-05-07 03:15:14,652 - INFO - ('127.0.0.1', 34446) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 03:15:14,659 - INFO - connection open
2025-05-07 03:15:14,662 - INFO - connection closed
2025-05-07 03:34:44,760 - INFO - ('127.0.0.1', 53110) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 03:34:44,782 - INFO - connection open
2025-05-07 03:34:44,786 - INFO - connection closed
2025-05-07 03:46:58,897 - INFO - ('127.0.0.1', 50122) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 03:46:58,928 - INFO - connection open
2025-05-07 03:46:58,933 - INFO - connection closed
2025-05-07 03:53:21,634 - INFO - ('127.0.0.1', 46304) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 03:53:21,643 - INFO - connection open
2025-05-07 03:53:21,649 - INFO - connection closed
2025-05-07 04:09:19,744 - INFO - ('127.0.0.1', 56686) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:09:19,746 - INFO - connection open
2025-05-07 04:09:19,749 - INFO - connection closed
2025-05-07 04:19:37,319 - INFO - ('127.0.0.1', 39068) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:19:37,326 - INFO - connection open
2025-05-07 04:19:37,333 - INFO - connection closed
2025-05-07 04:28:09,491 - INFO - ('127.0.0.1', 37876) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:28:09,493 - INFO - connection open
2025-05-07 04:28:09,495 - INFO - connection closed
2025-05-07 04:31:10,787 - INFO - ('127.0.0.1', 56420) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:31:10,802 - INFO - connection open
2025-05-07 04:31:10,807 - INFO - connection closed
2025-05-07 04:32:16,896 - INFO - ('127.0.0.1', 58608) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:32:16,898 - INFO - connection open
2025-05-07 04:32:16,900 - INFO - connection closed
2025-05-07 04:32:47,502 - INFO - ('127.0.0.1', 34406) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:32:47,504 - INFO - connection open
2025-05-07 04:32:47,505 - INFO - connection closed
2025-05-07 04:37:06,577 - INFO - ('127.0.0.1', 35722) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 04:37:06,592 - INFO - connection open
2025-05-07 04:37:06,617 - INFO - connection closed
2025-05-07 06:13:01,297 - INFO - ('127.0.0.1', 58344) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:13:01,315 - INFO - connection open
2025-05-07 06:13:01,533 - INFO - connection closed
2025-05-07 06:21:17,111 - INFO - ('127.0.0.1', 41804) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:21:17,117 - INFO - connection open
2025-05-07 06:21:17,125 - INFO - connection closed
2025-05-07 06:27:44,679 - INFO - ('127.0.0.1', 39640) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:27:44,686 - INFO - connection open
2025-05-07 06:27:44,691 - INFO - connection closed
2025-05-07 06:31:48,894 - INFO - ('127.0.0.1', 52308) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:31:48,899 - INFO - connection open
2025-05-07 06:31:48,901 - INFO - connection closed
2025-05-07 06:35:07,327 - INFO - ('127.0.0.1', 43552) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:35:07,342 - INFO - connection open
2025-05-07 06:35:07,350 - INFO - connection closed
2025-05-07 06:47:51,798 - INFO - ('127.0.0.1', 43036) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:47:51,809 - INFO - connection open
2025-05-07 06:47:51,820 - INFO - connection closed
2025-05-07 06:54:25,457 - INFO - ('127.0.0.1', 57800) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:54:25,462 - INFO - connection open
2025-05-07 06:54:25,466 - INFO - connection closed
2025-05-07 06:56:30,966 - INFO - ('127.0.0.1', 50740) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 06:56:30,968 - INFO - connection open
2025-05-07 06:56:30,969 - INFO - connection closed
2025-05-07 07:01:01,162 - INFO - ('127.0.0.1', 56810) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:01:01,184 - INFO - connection open
2025-05-07 07:01:01,203 - INFO - connection closed
2025-05-07 07:06:23,336 - INFO - ('127.0.0.1', 47364) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:06:23,352 - INFO - connection open
2025-05-07 07:06:23,358 - INFO - connection closed
2025-05-07 07:08:29,719 - INFO - ('127.0.0.1', 49578) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:08:29,728 - INFO - connection open
2025-05-07 07:08:29,733 - INFO - connection closed
2025-05-07 07:12:04,255 - INFO - ('127.0.0.1', 44580) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:12:04,264 - INFO - connection open
2025-05-07 07:12:04,268 - INFO - connection closed
2025-05-07 07:15:32,289 - INFO - ('127.0.0.1', 38736) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:15:32,298 - INFO - connection open
2025-05-07 07:15:32,302 - INFO - connection closed
2025-05-07 07:18:16,076 - INFO - ('127.0.0.1', 48598) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:18:16,078 - INFO - connection open
2025-05-07 07:18:16,080 - INFO - connection closed
2025-05-07 07:21:38,896 - INFO - ('127.0.0.1', 60738) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:21:38,919 - INFO - connection open
2025-05-07 07:21:38,929 - INFO - connection closed
2025-05-07 07:26:12,816 - INFO - ('127.0.0.1', 47134) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:26:12,823 - INFO - connection open
2025-05-07 07:26:12,827 - INFO - connection closed
2025-05-07 07:37:12,591 - INFO - ('127.0.0.1', 58264) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 07:37:12,602 - INFO - connection open
2025-05-07 07:37:12,611 - INFO - connection closed
2025-05-07 08:04:16,822 - INFO - ('127.0.0.1', 37970) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 08:04:16,831 - INFO - connection open
2025-05-07 08:04:16,836 - INFO - connection closed
2025-05-07 08:09:59,824 - INFO - ('127.0.0.1', 40746) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 08:09:59,832 - INFO - connection open
2025-05-07 08:10:00,042 - INFO - connection closed
2025-05-07 08:36:38,144 - INFO - ('127.0.0.1', 57678) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 08:36:38,152 - INFO - connection open
2025-05-07 08:36:38,156 - INFO - connection closed
2025-05-07 08:52:10,207 - INFO - ('127.0.0.1', 56352) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 08:52:10,218 - INFO - connection open
2025-05-07 08:52:10,225 - INFO - connection closed
2025-05-07 09:04:15,381 - INFO - ('127.0.0.1', 57562) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 09:04:15,388 - INFO - connection open
2025-05-07 09:04:15,393 - INFO - connection closed
2025-05-07 09:45:30,922 - ERROR - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 76, in collapse_excgroups
  |     yield
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 174, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 772, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    |     with recv_stream, send_stream, collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    |     response = await handler(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    |     raise app_exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    |     await route.handle(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    |     response = await f(request)
    |                ^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    |     return BaseResponse(e.args[0]["message"]).error()
    |                         ~~~~~~~~~^^^^^^^^^^^
    | TypeError: 'MaxRetryError' object is not subscriptable
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    with recv_stream, send_stream, collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    raise app_exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    return BaseResponse(e.args[0]["message"]).error()
                        ~~~~~~~~~^^^^^^^^^^^
TypeError: 'MaxRetryError' object is not subscriptable
2025-05-07 09:45:33,424 - ERROR - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 76, in collapse_excgroups
  |     yield
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 174, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 772, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    |     with recv_stream, send_stream, collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    |     response = await handler(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    |     raise app_exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    |     await route.handle(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    |     response = await f(request)
    |                ^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    |     return BaseResponse(e.args[0]["message"]).error()
    |                         ~~~~~~~~~^^^^^^^^^^^
    | TypeError: 'MaxRetryError' object is not subscriptable
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    with recv_stream, send_stream, collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    raise app_exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    return BaseResponse(e.args[0]["message"]).error()
                        ~~~~~~~~~^^^^^^^^^^^
TypeError: 'MaxRetryError' object is not subscriptable
2025-05-07 09:45:34,480 - ERROR - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 76, in collapse_excgroups
  |     yield
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 174, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/anyio/_backends/_asyncio.py", line 772, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    |     with recv_stream, send_stream, collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    |     response = await handler(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    |     raise app_exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    |     await route.handle(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    |     raise exc
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    |     response = await f(request)
    |                ^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    |     return BaseResponse(e.args[0]["message"]).error()
    |                         ~~~~~~~~~^^^^^^^^^^^
    | TypeError: 'MaxRetryError' object is not subscriptable
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/protocols/http/httptools_impl.py", line 409, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/uvicorn/middleware/proxy_headers.py", line 60, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/applications.py", line 1054, in __call__
    await super().__call__(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 187, in __call__
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/errors.py", line 165, in __call__
    await self.app(scope, receive, _send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 173, in __call__
    with recv_stream, send_stream, collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_utils.py", line 82, in collapse_excgroups
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 175, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/middleware.py", line 10, in logging_middleware
    response = await handler(request)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 153, in call_next
    raise app_exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/base.py", line 140, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/cors.py", line 85, in __call__
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 714, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 734, in app
    await route.handle(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 76, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/starlette/routing.py", line 73, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 301, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/venv_api/lib/python3.12/site-packages/fastapi/routing.py", line 212, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/manhhai01/Desktop/dev/deploy_3/hdbank-research-model_service-c9279bfc573f/api/domain/train/controller.py", line 53, in train
    return BaseResponse(e.args[0]["message"]).error()
                        ~~~~~~~~~^^^^^^^^^^^
TypeError: 'MaxRetryError' object is not subscriptable
2025-05-07 10:05:04,828 - INFO - ('127.0.0.1', 55410) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 10:05:04,844 - INFO - connection open
2025-05-07 10:05:04,847 - INFO - connection closed
2025-05-07 10:23:36,147 - INFO - ('127.0.0.1', 38860) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 10:23:36,156 - INFO - connection open
2025-05-07 10:23:36,160 - INFO - connection closed
2025-05-07 10:40:49,807 - INFO - ('127.0.0.1', 56696) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 10:40:49,817 - INFO - connection open
2025-05-07 10:40:49,821 - INFO - connection closed
2025-05-07 11:10:54,512 - INFO - ('127.0.0.1', 39246) - "WebSocket /train/ws?password=$2b$12$5LNCQ07.7rQnz.bnQauTFOKKGcR6sOqXpTfuoRyj7dIJSiglFWVXe" [accepted]
2025-05-07 11:10:54,534 - INFO - connection open
2025-05-07 11:10:54,539 - INFO - connection closed
